fireblocks_sdk/sdk.py

```diff
@@ -726,6 +727,35 @@ def get_exchange_accounts(self):
 
         return self._get_request("/v1/exchange_accounts")
 
+    def get_exchange_accounts_paged(self, paged_exchange_accounts_request_filters: PagedExchangeAccountRequestFilters=PagedExchangeAccountRequestFilters()):
```

Comment:
Mutable default arguments should be avoided. Please use `None` and add an if-statement inside the method

Reply:
remove it (should behave like 'get_vault_accounts_with_page_info')